receiver/jmxmetricreceiver/config_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package jmxmetricextension
+package jmxmetricreceiver
```

Comment:
nit: based on `jmx_metrics` name in the config, the package should be called `jmxmetricsreceiver` (i.e. plural).
Alternatively, we could just remove "metrics" completely and rename it to `jmx` in the config with a `jmxreceiver` package.

Reply:
@jrcamp @tigrannajaryan @bogdandrutu, thoughts?  I'm ok with `jmxreceiver`.

Reply:
I am fine with jmxreceiver, this is not very important :)

Reply:
Thanks, naming is always important imo, though I can get caught up in details...

Reply:
Either `jmxmetricreceiver` or `jmxreceiver` SGTM, no preference, choose what you like most.